src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3764,9 +3764,9 @@ static void window_ride_operating_invalidate(rct_window* w)
         if (ride->type == RIDE_TYPE_TWIST)
         {
             uint16_t arg;
-            std::memcpy(&arg, gCommonFormatArgs + 18, sizeof(uint16_t));
             ft = Formatter::Common();
             ft.Increment(18);
+            ft.CopyInto<uint16_t*>(&arg, 1);
```

Comment:
I don't like this. We should get the correct variable from earlier in the function not copy it from the global buffer.